Resources/Prototypes/_NF/Shipyard/nightlight.yml

```
@@ -12,7 +12,7 @@
   id: Nightlight
   name: FSB Nightlight
   description: The Far Star Biotech company is please to present the most forward facing testbed for the effects of bioregenerative nutrient on humanoid physiology to date! Bone Apple Teeth!
-  price: 43000 # I'm tired of fighting you appraisal bot tell me what you want.
+  price: 44500 # I'm tired of fighting you appraisal bot tell me what you want.
```

```suggestion
  price: 45000 # Botany bay
```
Changes from the Nightlight PR.
Values seem fine otherwise.